huff_cli/src/huffc.rs

```diff
@@ -83,14 +87,15 @@ struct Huff {
     #[clap(short = 'c', long = "constants", multiple_values = true)]
     constants: Option<Vec<String>>,
 
-    /// Test subcommand
     #[clap(subcommand)]
-    test: Option<TestCommands>,
+    sub: Option<Subcommands>,
 }
 
 #[derive(Subcommand, Clone, Debug)]
-enum TestCommands {
-    /// Test subcommand
+enum Subcommands {
```

Comment:
As this is now a subcommand, the current `huffc some/path/evm.huff` isn't working anymore. Any workaround ?